schemata_json_schema/src/Plugin/Type/TypeMapperPluginManager.php

```diff
@@ -34,7 +34,7 @@ class TypeMapperPluginManager extends DefaultPluginManager implements FallbackPl
    *   The module handler to invoke the alter hook with.
    */
   public function __construct(\Traversable $namespaces, ModuleHandlerInterface $module_handler) {
-    parent::__construct('Plugin/json_schema/type_mapper', $namespaces, $module_handler, 'Drupal\json_schema\Plugin\TypeMapperInterface', 'Drupal\json_schema\Annotation\TypeMapper');
+    parent::__construct('Plugin/type_mapper', $namespaces, $module_handler, 'Drupal\schemata_json_schema\Plugin\type_mapper\TypeMapperInterface', 'Drupal\schemata_json_schema\Annotation\TypeMapper');
```

Comment:
I don't think there is a prescription to namespaces for plugins. Two different plugin types can live under the same `type_mapper` namespace. They'll be told apart by the interface they implement.